pkgs/build-support/setup-hooks/wrap-gapps-hook.sh

```diff
@@ -36,9 +36,12 @@ wrapGAppsHook() {
   done
 
   if [ -z "$dontWrapGApps" ]; then
-    for i in $prefix/bin/* $prefix/libexec/*; do
-      echo "Wrapping app $i"
-      wrapProgram "$i" "${gappsWrapperArgs[@]}"
+    # Loop over files, without falling prey to filenames containing
+    # newlines, backslashes, or other horrible things.
+    find "${prefix}/bin" "${prefix}/libexec" -type f -executable -print0 \
+    | while IFS= read -r -d '' file; do
```

Comment:
Ok. Probably works. I was just reading this [FAQ](http://mywiki.wooledge.org/BashFAQ/020), where variables were set in the loop, which is not the case here.